R/new-dataset.R

```diff
@@ -65,7 +65,7 @@ newDatasetFromFile <- function (file, name=basename(file), ...) {
         halt("File not found")
     }
     ds <- createDataset(name=name, ...)
-    ds <- addSourceToDataset(ds, createSource(file))
+    ds <- addSourceToDataset(ds, createSource(file), savepoint=FALSE)
```

Comment:
What is `savepoint=FALSE`, and how does it relate to `autorollback` as an attribute on POST /batches/? Do we need two new attributes here?

It looks like you've added it everywhere there's a POST to /batches/ except for appending a dataset, where instead there's autorollback=TRUE. If it's important that they be distinct, then maybe savepoint=FALSE should be the default (in these functions) so that it's not passed around everywhere.